bot_cli/__main__.py

```diff
@@ -36,6 +36,10 @@ def main(address_book):
                 print_info("How can I help you?")
             case "add-contact":
                 commands.add_contact(args)
+            case "add-address":
```

Comment:
I want to have a `change-email` and `change-address` etc commands. The user of the CLI should not be concerned with the implementation detail that it is some dictionary and is updated anyway. To be very honest, I want `add-address` to fail the second time address is added because it's how from the point of view of the user, how the system should behave, unless it stores more than one address.

Reply:
In my opinion it's redundant in scope of the ptoject. The default behaviot matches the customer's expectation (that's the main goal of the project).
Won't do.